database/index.js

```diff
@@ -0,0 +1,13 @@
+var mysql = require('mysql');
+
+var connection = mysql.createConnection({
+  user: 'root',
+  password: '',
+  database: 'overview'
+});
+
+connection.connect();
```

Comment:
looks great! Nothing wrong with line 9, but just an observation that I don't think it's necessary since with mySQL npm module, the query implicitly opens the connection -- "However, a connection can also be implicitly established by invoking a query:" https://www.npmjs.com/package/mysql